packages/db-objects-redis/src/lib/objects/objectsUtils.ts

```diff
@@ -329,41 +333,42 @@ export function checkFileRights(
      return callback(null, options);*/
 }
 // For users and groups
-function getDefaultAdminRights(acl?: Record<string, any>, _isState?: boolean) {
-    acl = acl || {};
-    acl.file = {
-        list: true,
-        read: true,
-        write: true,
-        create: true,
-        delete: true
-    };
-    acl.object = {
-        create: true,
-        list: true,
-        read: true,
-        write: true,
-        delete: true
-    };
-    acl.users = {
-        create: true,
-        list: true,
-        read: true,
-        write: true,
-        delete: true
-    };
-    acl.state = {
-        read: true,
-        write: true,
-        delete: true,
-        create: true,
-        list: true
+function getDefaultAdminRights(acl?: ioBroker.ObjectPermissions, _isState?: boolean) {
+    return {
+        ...acl,
+        file: {
+            list: true,
+            read: true,
+            write: true,
+            create: true,
+            delete: true
+        },
+        object: {
+            create: true,
+            list: true,
+            read: true,
+            write: true,
+            delete: true
+        },
+        users: {
+            create: true,
+            list: true,
+            read: true,
+            write: true,
+            delete: true
+        },
+        state: {
+            read: true,
+            write: true,
+            delete: true,
+            create: true,
+            list: true
+        }
     };
-
-    return acl;
 }
 
-export function getUserGroup(objects: any, user: string, callback: () => void) {
+// FIXME: Not sure about the expected return type here? ioBroker.GroupObject maybe? Or an array thereof?
```

Comment:
This should have a clear return type, but I currently can't decipher it.

Reply:
I've added it from my understanding